addons/info/src/components/stories/mock-data/summary.md.js

```diff
@@ -0,0 +1,300 @@
+export default `
+
```

Comment:
That is some regression test, I like it!

Reply:
Thanks)